### ISSUE TITLE
Join Brain.js as one of the official visualizers 

### ISSUE THREAD

**Issue by robertleeplummerjr** [marcusbuffett/Three.js-Neural-Network-Visualizer#2]
Because it is simply a fantastic tool!

**Comment by robertleeplummerjr** [comment]
Given any thought to this with the work that has been done here: https://github.com/marcusbuffett/Three.js-Neural-Network-Visualizer/issues/1?

**Comment by marcusbuffett** [comment]
Yeah of course, this would be awesome. I've upgraded to the latest of brain.js now, just let me know if you need anything more from me.
